### PR TITLE
bitnami/mysql: add values.schema.json check

### PR DESCRIPTION
**Pull request by AndrewDi**
**Description of the change**

add values.schema.json for mysql chart

**Benefits**

user can choose parameters base from values.schema.json

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/mysql/values.schema.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
+{
```
Comment (marcosbc) [inline bitnami/mysql/values.schema.json:1]: It seems like this file is rendered with a 4-space indentation. In this repo we're currently following 2-space indentation for JSON files (i.e. WordPress [`values.schema.json`](https://github.com/bitnami/charts/blob/master/bitnami/wordpress/values.schema.json#L26)). Could you fix this? I'd recommend formatting with the `jq` CLI tool.

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please have a look at my comments and let me know if you have any questions.

File: bitnami/mysql/values.schema.json [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                        "runAsUser": {
+                            "type": "integer",
+                            "default": 1001,
+                            "hidden": {
+                                "value": false,
+                                "path": "primary/containerSecurityContext/enabled"
+                            }
+                        }
+                    }
+                },
```
[the remark this answers is not in the document]
Comment (AndrewDi) [reply bitnami/mysql/values.schema.json:85]: I have test it on Openshift Platform, default `podSecurityContext`/`containerSecurityContext` from values.yml will fail to create pod, so I expect user can choose to disable it by menu.

File: bitnami/mysql/values.schema.json [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                "persistence": {
+                    "type": "object",
+                    "title": "Enable persistence using Persistent Volume Claims",
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "default": true,
+                            "title": "If true, use a Persistent Volume Claim, If false, use emptyDir"
+                        },
```
[the remark this answers is not in the document]
Comment (AndrewDi) [reply bitnami/mysql/values.schema.json:94]: I have test it on Openshift Platform, default `podSecurityContext`/`containerSecurityContext` from values.yml will fail to create pod, so I expect user can choose to disable it by menu.

File: bitnami/mysql/values.schema.json (hunk 1 above)
Comment (AndrewDi) [reply bitnami/mysql/values.schema.json:1]: done

**Review by marcosbc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**